plugins/modules/pnp_intent.py

```diff
@@ -78,7 +81,8 @@
         description: Vlan Id allocated for claimimg of Wireless Controller
         type: str
       ip_interface_name:
-        description: Name of the Interface used for Pnp by the Wireless Controller
+        description: Name of the Interface used for Pnp by the Wireless Controller. It should be configured on the Controller
```

Comment:
description: Specifies the interface name utilized for Plug and Play (PnP) by the Wireless Controller. Ensure this interface is pre-configured on the Controller prior to device claiming.

Reply:
Done